src/main/java/domain/policy/ExpellState.java

```diff
@@ -0,0 +1,31 @@
+package domain.policy;
+
+import java.util.Arrays;
+
+public enum ExpellState {
+    EXPELL("제적", 6),
+    INTERVIEW("면담", 3),
+    WARNING("경고", 2),
+
+    NONE("정상출석", 0);
+
+    private static final int LATE_ABSENT_RATE = 3;
+
+    public String description;
+    private int limit;
```

Comment:
바뀌지 않을 필드값엔 final 키워드를 붙여주면 좋습니다. 😄 
```suggestion
    public final String description;
    private final int limit;
```

Reply:
수정하겠습니다~!